.github/workflows/test.yml

```diff
@@ -175,6 +176,10 @@ jobs:
         env:
           ZEBRA_TEST_LIGHTWALLETD: '1'
 
+  # Regenerate mandatory checkpoint disks.
+  # Runs:
+  # - on every PR update, but only if Zebra's state version changes in the PR
+  # - on request
```

Comment:
👍

Reply:
Please avoid adding comments to an approved PR, because it cancels the merge.